collectors/charts.d.plugin/nvme/nvme.chart.sh

```diff
@@ -0,0 +1,140 @@
+# netdata
```

Comment:
```suggestion
# shellcheck shell=bash
# no need for shebang - this file is loaded from charts.d.plugin
# SPDX-License-Identifier: GPL-3.0-or-later

# netdata
```

This will resolve the shellcheck error here, as well as adding the small bit of other boilerplate we need at the top of the file.